dinhthuankhang/19_10/com.example.milk_handle/Handle.kt

```diff
@@ -0,0 +1,11 @@
+package com.example.milk_handle
+
+data class Handle(val temperature: Int,
+                  val time: IntRange,
+                  val userObject: String,
+                  val expiry: String)
+open class TypeOfMilk {
```

Comment:
Type of milk nên để thành enum thì hơn. Và class này của e ko có class nào kế thừa nó, nên ko cần để `open` cũng đc